vector-app/src/gplay/AndroidManifest.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
This Manifest is not existing in Tchap-Android
The actual one is: vector/src/gplay/AndroidManifest.xml

Reply:
the modularization has been included in EA-v1.4.36, that's why we don't have it yet

Reply:
We should move the firebase entries from this file to vector/src/gplay/AndroidManifest